JeongJaeHeon/app.js

```diff
@@ -0,0 +1,42 @@
+const http = require('http');
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+const dotenv = require('dotenv');
+
+dotenv.config()
+
+const { DataSource } = require('typeorm')
+
+const myDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE,
+})
+
+myDataSource.initialize()
+    .then(() => {console.log("Data Source has been initialized")})
+    .catch(() => {console.log("Data Source has not been initialized")})
+
+app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan('combined'));
+
+// health check
+app.get('/ping',(req, res, next) => {
+    res.status(201).json({message : 'pong'});
+});
+
+const server = http.createServer(app);
+const PORT = process.env.PORT;
+
+const start = async() => {
+    server.listen(PORT, () => console.log(`Server is listening to ${PORT}`));
+}
```

Comment:
express 미들웨어를 통해 `app` 자체로도 서버를 동작할 수 있기 때문에 `http built-in` 모듈을 사용하지 않아도 되는 부분 같습니다!

아래와 같이 수정하여 `app.js`가 정상적으로 실행되는지 다시 한번 확인해주세요!

```suggestion
const PORT = process.env.PORT;

const start = async() => {
    app.listen(PORT, () => console.log(`Server is listening to ${PORT}`));
}
```